src/compiler/types.ts

```diff
@@ -981,13 +981,19 @@ namespace ts {
         multiLine?: boolean;
     }
 
+    export type EntityNameExpression = Identifier | PropertyAccessEntityNameExpression;
+    export type EntityNameOrEntityNameExpression = EntityName | EntityNameExpression;
+
     // @kind(SyntaxKind.PropertyAccessExpression)
     export interface PropertyAccessExpression extends MemberExpression, Declaration {
         expression: LeftHandSideExpression;
         name: Identifier;
     }
-
-    export type IdentifierOrPropertyAccess = Identifier | PropertyAccessExpression;

```

Comment:
I'm not sure, this is a better question for @vladima

---
In reply to: [72693336](https://github.com/Microsoft/TypeScript/pull/10013#discussion_r72693336) [](ancestors = 72693336)